install_config/topics/configuring_for_gce_registry.adoc

```diff
@@ -0,0 +1,284 @@
+available in GCP.
+====
+
+See the link:https://cloud.google.com/storage/docs/creating-buckets[Creating storage buckets documentation] for more information.
+
+== Configuring {product-title} registry for GCP using Ansible
+
+.Procedure
+
+To configure the Ansible inventory for the registry to use a Google Cloud
+Storage (GCS) bucket:
+
+----
+[OSEv3:vars]
+# GCP Provider Configuration
+openshift_hosted_registry_storage_provider=gcs
+openshift_hosted_registry_storage_kind=object
+openshift_hosted_registry_replicas=1 <1>
+openshift_hosted_registry_storage_gcs_bucket=<bucket_name> <2>
+openshift_hosted_registry_storage_gcs_keyfile=<bucket_keyfile> <3>
+openshift_hosted_registry_storage_gcs_rootdirectory=<registry_directory> <4>
+----
+<1> The number of replicas to configure.
+<2> The bucket name to be used as registry storage.
+<3> The path on the installer host where the bucket's keyfile is located if
+using a custom key file to encrypt the data.
+<4> Directory used to store the data. `/registry` by default
+
+For more information, see link:https://cloud.google.com/storage/docs/[Cloud Storage in the GCP documentation].
+
+== Manually configuring {product-title} registry for GCP
+
+To use GCP object storage, edit the registry’s configuration file and mount to the registry pod.
+
+See the https://docs.docker.com/registry/storage-drivers/gcs/[Google Cloud Storage Driver documentation] for more information about storage driver configuration files.
+
+.Procedure
+
+. Export the current *_config.yml_*:
++
+[source,bash]
+----
+$ oc get secret registry-config \
+    -o jsonpath='{.data.config\.yml}' -n default | base64 -d \
+  >> config.yml.old
+----
+
+. Create a new configuration file from the old *_config.yml_*:
++
+[source,bash]
+----
+$ cp config.yml.old config.yml
+----
+
+. Edit the file to include the GCP parameters. Specify the bucket and keyfile in the `storage` section of a registry’s configuration file:
++
+[source,yaml]
+----
+storage:
+  delete:
+    enabled: true
+  cache:
+    blobdescriptor: inmemory
+  gcs:
+    bucket: ocp-registry <1>
+    keyfile: mykeyfile <2>
+----
+<1>	Replace with the GCP bucket name.
+<2> A private service account key file in JSON format. If using the Google Application Default Credentials, do not specify the `keyfile` parameter.
+
+. Delete the `registry-config` secret:
++
+[source,bash]
+----
+$ oc delete secret registry-config -n default
+----
+
+. Recreate the configmap to reference the updated configuration file:
++
+[source,bash]
+----
+$ oc create secret generic registry-config \
+    --from-file=config.yml -n default
+----
+
+. Redeploy the registry to read the updated configuration:
++
+[source,bash]
+----
+$ oc rollout latest docker-registry -n default
+----
+
+=== Verify the registry is using GCP object storage
+
+To verify if the registry is using GCP bucket storage:
+
+.Procedure
+
+. After a successful registry deployment, the registry `deploymentconfig` does
```

Comment:
So, even if you configure the docker registry to use gcp cloud storage, the volume is still there.

Those steps are to show how to effectively see if it is using the gcp cloud storage or not because from the OCP standpoint (so oc commands), the docker registry is still using a volume.

I'd say:

'After a successful registry deployment using GCP storage, the registry `deploymentconfig` does'